src/server/pps/server/defaults_test.go

```diff
@@ -194,6 +194,15 @@ func TestAPIServer_SetClusterDefaults_regenerate(t *testing.T) {
 		ClusterDefaultsJson: `{"create_pipeline_request": {"datum_tries": 17, "autoscaling": true}}`,
 	})
 	require.NoError(t, err, "SetClusterDefaults must succeed")
+	type projectDefaultsSetter interface {
```

Comment:
is this necessary?

Reply:
I think now that `SetProjectDefaults` is an RPC, I can eliminate this.